src/panel/direction/RoadMapItem.jsx

```diff
@@ -3,11 +3,11 @@ import classnames from 'classnames';
 
 import { Divider, Flex } from 'src/components/ui';
 
-const RoadMapItem = ({ children, icon, distance, className, line, ...rest }) =>
+const RoadMapItem = ({ children, icon, distance, className, line, flex, ...rest }) =>
```

Comment:
`flex` isn't a good param name, we don't understand what it does. `alignItems` would be better.
And you could give it a default value equals to `center`, that way you don't need to modify other files not concerned by this PR and manage the special step-by-step case as a kind of exception (which it is).